src/_main_/utils/pagination/__init__.py

```diff
@@ -0,0 +1,25 @@
+from django.core.paginator import Paginator, EmptyPage
+
+from _main_.utils.common import serialize_all
+PAGINATION_LIMIT = 50
```

Comment:
could we pass the page size based on the Admin portal settings page length (can display up to 100 in the lists)